R/build_data.R

```diff
@@ -1,15 +1,40 @@
-# runs on package build. Need to fix for windows...
-if (file.exists('~/Documents/R/mda.streams/inst/init_data.R')){
-  source('~/Documents/R/mda.streams/inst/init_data.R')
-}
-
+#' Variable code data
+#'
+#' Variable code data
+#'
+#' @name var_codes
+#' @docType data
+#' @export
+NULL
 
-#' State code data
+#' Source code data
 #'
-#' State code data
+#' Source code data
 #'
-#' @name states
+#' @name src_codes
 #' @docType data
-#' @export states
-#' @keywords state codes
-NULL
\ No newline at end of file
+#' @export
+NULL
+
+#' For use in package development only: build R/sysdata.rda
+#' 
+#' Moves/reformats the tables of variable and source metadata from inst/extdata 
+#' (where they can be manually edited in .tsv format) to R/sysdata.rda (the 
+#' storage location for data used internally by mda.streams)
+#' 
+#' @keywords internal
+build_sysdata <- function() {
```

Comment:
so this is manually called? I always wondered about the "right" way to do something like this when you want to have a sys.data built on the fly during the build process.